src/Normalizer/ObjectNormalizer.php

```diff
@@ -78,7 +78,7 @@ public function normalize($object, $format = null, array $context = array())
      */
     public function supportsNormalization($data, $format = null)
     {
-        return $data instanceof Object;
+        return $data instanceof StatementObject;
```

Comment:
For backwards compatibility, this should be `return $data instanceof Object || $data instanceof StatementObject;`.